tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -45,6 +48,7 @@ Define {
 	DMIC1_NAME			'NoCodec-7'
 	DMIC0_PCM_CAPS			'Passthrough Capture 13'
 	DMIC0_PIPELINE_STREAM_NAME	'copier.DMIC.14.1'
+	DMIC1_PCM_CAPS			'Capture 18'
```

Comment:
Capture 18? Can we call this "WoV Capture" instead?

Reply:
DMIC1 is a general usage, it can be used for other purposes.